app/types/TransactionLog.d.ts

```diff
@@ -21,25 +23,69 @@ export interface TransactionLog {
   finalizedBlockIndex: StringUInt64 | null;
   inputTxoIds: StringHex[]; // FK this is gone?
   inputTxos: TransactionAbbreviation[];
-  isSentRecovered: boolean | null;
   object: 'transaction_log';
   offsetCount: number;
   outputTxoIds: StringHex[]; // FK this is gone?
   outputTxos: TransactionAbbreviation[];
-  recipientAddressId: StringB58 | null; // FK this is gone?
+  recipientAddressId: StringB58;
   sentTime: string | null; // FIX-ME: Confirm type
-  status:
-    | 'tx_status_built'
-    | 'tx_status_pending'
-    | 'tx_status_received'
-    | 'tx_status_succeded'
-    | 'tx_status_failed';
+  status: Status;
   submittedBlockIndex: StringUInt64 | null;
   transactionLogId: StringHex;
   valuePmob: StringUInt64;
 }
 
+type Status =
+  | 'tx_status_built'
+  | 'tx_status_pending'
+  | 'tx_status_received'
+  | 'tx_status_succeded'
+  | 'tx_status_failed'
+  | 'tx_status_unspent'
+  | 'tx_status_spent';
```

Comment:
going back the `default` statement where this is used, we should have a `tx_status_unknown` value